src/Dialogs/Imap/ImapDialog.vala

```diff
@@ -161,6 +176,17 @@ public class OnlineAccounts.ImapDialog : Hdy.Window {
         smtp_encryption_combobox.append ("starttls-on-standard-port", "STARTTLS");
         smtp_encryption_combobox.active = 2;
 
+        var smtp_auth_label = new Gtk.Label (_("Authentication:")) {
+            xalign = 1
+        };
+
+        smtp_auth_combobox = new Gtk.ComboBoxText () {
```

Comment:
This should probably be packed in with the Username and Password fields so that it follows the "Use IMAP credentials" and "No auth required" settings right?